brave/browser-laptop
# Changed share l10n string to use a generic template for each share menu items

Fix #8604

Submitter Checklist:


Test Plan:
1. Build from the source: npm run watch
2. Run Brave: npm run start
3. In the URL bar type: https://www.brave.com/
4. On the menu bar click on File.
5. Under File look for Share.
6. Under Share click on one item with share page.

Reviewer Checklist:


Tests
